types/kagekiri.d.ts

```diff
@@ -1,3 +1,4 @@
 // Type definitions for kagekiri
 export function querySelector(selector: string, context?: Node): HTMLElement | null;
```

Comment:
`querySelector` should return an `Element`, not an `HTMLElement`. It's possible to query an SVG here and get an `SVGElement`.